### PR TITLE
IGVF-2237-fix-IGSR

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

**Review by forresttanaka — APPROVED** [review]

**Comment by ottojolanki** [comment]
Looks like the demo backend url needs to be removed from the config before merging.

**Comment by forresttanaka** [comment]
I’ll wait for tests to finish and then merge it.

**Pull request merged**
